.github/CODEOWNERS

```diff
@@ -1 +1,13 @@
-* @hashicorp/nomad-eng
+# Each line is a file pattern followed by one or more owners.
+# More on CODEOWNERS files: https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners
+
+# Default owner
```

Comment:
I think perhaps we can lose all the boilerplate comments here?

Reply:
We are using this as a standard template for CODEOWNERS files for all repos under our ownership.